src/olympia/files/utils.py

```diff
@@ -251,14 +255,18 @@ def apps(self):
         )
 
         for app, default_min_version in apps:
-            strict_min_version = (
-                # At least this version supports installing.
-                get_simple_version(self.gecko.get('strict_min_version')) or
-                default_min_version)
+            if self.guid is None:
+                strict_min_version = amo.DEFAULT_WEBEXT_MIN_VERSION_NO_ID
```

Comment:
Outch, I feel like it's Monday. Yeah, that's not supposed to happen. Let
me fix that :)
 
 
On Thu, Jun 2, 2016, at 03:22 PM, Mathieu Pillard wrote:
> In src/olympia/files/utils.py[1]:


> > @@ -251,14 +255,18 @@ def apps(self): >          ) > >          for
> > app, default_min_version in apps: > -            strict_min_version
> > = ( > -                # At least this version supports installing.
> > > -                get_simple_version(self.gecko.get('strict_min_ve-
> > rsion')) or > -                default_min_version) > +
> > if self.guid is None: > +                strict_min_version =
> > amo.DEFAULT_WEBEXT_MIN_VERSION_NO_ID
>
> So, if no id is given, you're overriding the strict_min_version no
> matter what ? What if it was specifying a higher version ? It may not
> matter right now because we don't have releases past 48.0 yet, but
> this could be a problem in the future.
> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[2], or mute the thread[3].
>
 

Links:

  1. https://github.com/mozilla/addons-server/pull/2832#discussion_r65537506
  2. https://github.com/mozilla/addons-server/pull/2832/files/25d5c27695a433d949fbbbdd5eda70d306eed206#r65537506
  3. https://github.com/notifications/unsubscribe/AAIfGeOhUDlU3ZLLViuGOrZt21I3NxRmks5qHtj8gaJpZM4IsfSH